src/relay/pass/device_annotation.cc

```diff
@@ -485,7 +485,52 @@ class DeviceInfo {
 
 Expr RewriteAnnotatedOps(const Expr& expr, int fallback_device) {
   RewriteAnnotation rewrote = RewriteAnnotation();
-  return rewrote.Rewrite(expr, fallback_device);
+  Expr new_expr = rewrote.Rewrite(expr, fallback_device);
+
+  // Remove OnDevice operators. Note that this operators are only present at the
```

Comment:
```suggestion
  // Remove OnDevice operators. Note that these operators are only present at the
```
Otherwise, looks good to me.